css/fonts.styl

```diff
@@ -0,0 +1,8 @@
+@font-face {
```

Comment:
There is now a kuoto swiss mixin for this: http://kouto-swiss.io/#mixins_font-face